runelite-client/src/main/java/net/runelite/client/plugins/fishing/FishingPlugin.java

```diff
@@ -335,4 +347,39 @@ public void onVarbitChanged(VarbitChanged event)
 			trawlerNotificationSent = false;
 		}
 	}
+
+	@Subscribe
+	public void onWidgertLoaded(WidgetLoaded event)
+	{
+		if (event.getGroupId() == WidgetID.FISHING_TRAWLER_GROUP_ID)
+		{
+			endTime = Instant.now().plusSeconds(fishingTrawlerTimeLimit);
```

Comment:
As I said above, this should be start time and you should assign just Instant.now here

Reply:
If I dont assign the .plusSeconds() when will I manually add the timer?